documentation/Arbitraries.md

```diff
@@ -43,6 +43,7 @@ fc.assert(
 - [Combinators](#combinators)
   - [Simple](#simple)
   - [Array](#array)
+  - [Typed Array](#typed-array)
```

Comment:
Should it be in Combinators section? Might be better in Numeric as it does not offer any Combinators capabilities 🤔